BankChain/app/src/androidTest/java/nl/tudelft/ewi/ds/bankchain/bank/bunq/BunqTest.java

```diff
@@ -35,9 +35,9 @@ public void testBadUrl() throws Exception {
         Environment env;
 
         env = new Environment();
-        env.bank = "Bunq";
-        env.url = "https://example.com/";
-        env.apiKey = "";
+        env.setBank("Bunq");
+        env.setBankUrl("https://example.com/");
+        env.setBankApiKey("");
```

Comment:
this will crash as the api key cant bey empty however it should be filled locally (maybe put some comments about that)